src/lib/tecs/TECS.cpp

```diff
@@ -574,12 +574,12 @@ float TECSControl::_calcThrottleControlOutput(const STERateLimit &limit, const C
 
 	if (ste_rate.setpoint >= FLT_EPSILON) {
 		// throttle is between trim and maximum
-		throttle_predicted = param.throttle_trim + ste_rate.setpoint / limit.STE_rate_max *
+		throttle_predicted = param.throttle_trim_adjusted + ste_rate.setpoint / limit.STE_rate_max *
 				     (param.throttle_max - param.throttle_trim);
```

Comment:
This could be a bit confusing to people, wondering why the unadjusted trim is still being used. Maybe a comment here to clarify the STE rate component of the predicted throttle is more about climbing adjustment, and tuned about the raw trim value. Then the airspeed scheduling is done independently.

Ofc, this does make me wonder if that really makes sense to split them, or also adjust the trim center for the energy compensation. (similar discussion also to what @RomanBapst mentioned on adjusting the max/min throttle for density + weight)

So alternative here basically adjusting trim throttle, min throttle, and max throttle all in your throttle calculation method outside of tecs, and just letting tecs know what it's got to work with.

Reply:
okay done!